frontend/ui/widget/menu.lua

```diff
@@ -898,7 +898,12 @@ function Menu:updatePageInfo(select_number)
             self.selected = { x = 1, y = select_number }
         end
         -- update page information
-        self.page_info_text:setText(util.template(_("page %1 of %2"), self.page, self.page_num))
+        -- don't show page x of x if there is only one page
+        if self.page_num > 1 then
+          self.page_info_text:setText(util.template(_("page %1 of %2"), self.page, self.page_num))
```

Comment:
Indentation.

Also weren't you proposing to make it Page or was there an objection to that?

```suggestion
            self.page_info_text:setText(util.template(_("Page %1 of %2"), self.page, self.page_num))
```

Reply:
In some of the UI it is “page” and in other areas it’s “Page”, will fix that up tomorrow